CRM/Contact/Form/Edit/Address.php

```diff
@@ -164,6 +164,9 @@ public static function buildQuickForm(&$form, $addressBlockCount = NULL, $sharin
       // shared address
       $form->addElement('checkbox', "address[$blockId][use_shared_address]", NULL, ts('Use another contact\'s address'));
 
+      // do we want to update employer for shared address
+      $form->addElement('checkbox', "address[$blockId][update_current_employer]", NULL, ts('Set this organization as current employer'));
```

Comment:
Switch this to `advcheckbox` to always get a value back instead of undefined for unchecked.

Reply:
@samuelsov ^^ looks this is currently blocking

Reply:
@colemanw @eileenmcnaughton ok thanks, i was not aware of advcheckbox alternative.